django-stubs/contrib/messages/test.pyi

```diff
@@ -0,0 +1,9 @@
+from typing import Any
+
+from django.contrib.messages.storage.base import Message
+from django.http.response import HttpResponse
+
+class MessagesTestMixin:
+    def assertMessages(
+        self, response: HttpResponse, expected_messages: list[dict[str, Any] | type[Message]], *, ordered: bool = ...
```

Comment:
I am not sure that this is correct. `expected_messages` should be equal to `list(get_messages(...))` result: https://github.com/django/django/blob/9a27c76021f934201cccf12215514a3091325ec8/django/contrib/messages/test.py#L6

`get_messages()` definition: https://github.com/typeddjango/django-stubs/blob/b0858a799b2cc7e9abec1e05ec5c0ce2c1875320/django-stubs/contrib/messages/api.pyi#L16

Reply:
I see. I'll fix it and upload it :)